File: src/main/java/at/yawk/wm/tac/TacConfig.java
package at.yawk.wm.tac;

import at.yawk.wm.style.FontDescriptor;
import at.yawk.wm.style.FontStyle;
import java.awt.*;
import lombok.Data;

/**
 * @author yawkat
 */
@Data
public class TacConfig {
    private FontDescriptor fontPrimary;
    private FontDescriptor fontPrimarySelected;
    private FontDescriptor fontSecondary;
    private FontDescriptor fontSecondarySelected;

    private Color colorBackground;
    private Color colorSelected;

    private int width;
    private int rowHeight;
    private int padding;
}

File: src/main/java/at/yawk/wm/tac/TacConfig.kt
package at.yawk.wm.tac

import at.yawk.wm.style.Color
import at.yawk.wm.style.StyleConfig

/**
 * @author yawkat
 */
object TacConfig {
    val fontPrimary = StyleConfig.tacPrimary
    val fontPrimarySelected = StyleConfig.tacPrimarySelected
    val fontSecondary = StyleConfig.tacSecondary
    val fontSecondarySelected = StyleConfig.tacSecondarySelected

    val colorBackground = Color.Solarized.base03
    val colorSelected = Color.Solarized.base02

    const val width = 400
    const val rowHeight = 22
    const val padding = 3
}
